src/gol.js

```diff
@@ -1,34 +1,33 @@
+export function countLiveNeighbours(board, i, j) {
```

Comment:
Good order of functions definitions. It is really good practice to define function before they will be used.